### PR TITLE
chore(litmus-portal): Added parallel and serial sequencing of experiments for workflow creation

### PR DESCRIPTION
**Pull request by amityt**
Signed-off-by: Amit Kumar Das <amit@chaosnative.com>

## Proposed changes

This PR adds:
- Sequencing functionality of experiments during workflow creation

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

## Special notes for your reviewer:

**1. Parallel sequencing for Kube-Proxy-Chaos experiement (Pre-defined experiment)**
Screenshot from 2021-04-21 20-00-15

**2. Parallel sequencing with custom workflows**
Screenshot from 2021-04-21 20-01-25

### CODE REVIEW

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts [hunk 1]
```diff
@@ -61,6 +61,7 @@ const useStyles = makeStyles((theme) => ({
   // Modal
   closeButton: {
     borderColor: theme.palette.border.main,
+    marginTop: theme.spacing(-3.75),
```
Comment (ishangupta-ds) [inline litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts:64]: create an issue also in litmus-ui for this negative margin

**Review by ishangupta-ds — APPROVED** [review]
LGTM

File: litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts (hunk 1 above)
Comment (amityt) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts:64]: Sure. I will try to remove this negative margin with some fix. Thanks @ishangupta-ds

Comment (ishangupta-ds) [reply litmus-portal/frontend/src/views/CreateWorkflow/TuneWorkflow/styles.ts:64]: No it's fine here, no need to deviate from the design but please create a issue or add link to this PR to exiting modal issue on litmus-ui

**Review by S-ayanide — APPROVED** [review]
LGTM

**Pull request merged**